### PR TITLE
Add port 5060, 5061 to blocked port list - fixes #1108

### LINKED ISSUE #1108: Add port 5060 to bad ports (SIP)
Packets on port 5060 are treated specially by some NAT devices. It may be possible to trick them with a carefully-crafted HTTP request body.

See https://samy.pl/slipstream.

### PR DESCRIPTION
**Pull request by mozfreddyb**
- [x] At least two implementers are interested (and none opposed):
   * [Firefox intent to ship thread](https://groups.google.com/d/msg/mozilla.dev.platform/FL-zKm4T6SA/vEUPOg7EAAAJ)
   * [Blink intent to ship](https://groups.google.com/a/chromium.org/g/blink-dev/c/0d4L4zZWcH4)
- [x] [Tests](https://github.com/web-platform-tests/wpt) are written and can be reviewed and commented upon at:
   * Part of the Gecko patch linked below
- [x] [Implementation bugs](https://github.com/whatwg/meta/blob/master/MAINTAINERS.md#handling-pull-requests) are filed:
   - Chrome:  https://bugs.chromium.org/p/chromium/issues/detail?id=1145680
   - Firefox: https://bugzilla.mozilla.org/show_bug.cgi?id=1674735
   - Safari:  https://bugs.webkit.org/show_bug.cgi?id=218557

***
<a href="https://whatpr.org/fetch/1109.html" title="Last updated on Nov 5, 2020, 8:08 AM UTC (e740c8b)">Preview</a> | <a href="https://whatpr.org/fetch/1109/65ec6e9...e740c8b.html" title="Last updated on Nov 5, 2020, 8:08 AM UTC (e740c8b)">Diff</a>

### CODE REVIEW

File: fetch.bs [hunk 1]
```diff
@@ -2278,6 +2278,8 @@ run these steps:
  <tr><td>2049<td>nfs
  <tr><td>3659<td>apple-sasl
  <tr><td>4045<td>npp
+ <tr><td>5060<td>sip
+ <tr><td>5061<td>sip+tls
```
Comment (annevk) [inline fetch.bs:2282]: sips (per https://www.iana.org/assignments/service-names-port-numbers/service-names-port-numbers.txt)

Comment (mozfreddyb) [reply fetch.bs]: thanks!

**Comment by ricea** [comment]
lgtm! Should I send something to blink-dev@chromium.org to officially indicate interest? If not, you can consider this comment as showing interest for Google Chrome.

**Comment by annevk** [comment]
I think this is fine given that it's a small change. I'll wait a bit for Youenn to comment before merging.

**Comment by youennf** [comment]
Seems good as well.
I filed https://bugs.webkit.org/show_bug.cgi?id=218557

**Review by ricea — APPROVED** [review]

**Comment by ricea** [comment]
I've filed the issue at https://bugs.chromium.org/p/chromium/issues/detail?id=1145680 but unfortunately it's view restricted. I'm trying to get the view restriction removed.

**Comment by ricea** [comment]
I sent an FYI to blink-dev: https://groups.google.com/a/chromium.org/g/blink-dev/c/0d4L4zZWcH4

**Comment by annevk** [comment]
Thanks all!

**Comment by lgrahl** [comment]
Technical questions: Why block it for HTTPS traffic?

**Comment by annevk** [comment]
There's no real reason to assume the server would deal better with HTTP over TLS I think. If HTTP is negotiated via ALPN that would be different, but thus far we haven't made that distinction and I'm not sure it's worth introducing it for this. Could be interesting in the future though if there are good reasons to have less port restrictions.

**Pull request merged**